### DISCUSSION TITLE
Locked and limited conversation to collaborators

### DISCUSSION THREAD

**Discussion by uazo** [bromite/bromite#1589] [Q&A] [answered]
@csagan5  just to show you, it is limited to me too (#1560)

![image](https://user-images.githubusercontent.com/29201891/142158085-c1f61387-14e9-48af-918a-a2f7b2ca2124.png)

**Comment by csagan5** [comment, marked as answer]
Yes, it works as intended; GitHub does not have good permissions control with collaborators.
